install/0000_99_machine-api-operator_00_tombstones.yaml

```diff
@@ -1,67 +1,76 @@
-apiVersion: cloudcredentials.openshift.io/v1
+apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
   name: openshift-machine-api
   namespace: openshift-cloud-credential-operator
   annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
     release.openshift.io/delete: "true"
 ---
-apiVersion: apiextensions.k8s.io
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I think this needs to be `v1`, the `v1beta1` API was deprecated and possibly even removed

Reply:
thanks, i'll give that a try. i used the version from the original manifest where it was created, with the thinking that it would need to match, but what you are suggesting makes sense.